From: Markus Koschany <apo@debian.org>
Date: Sun, 9 Dec 2018 14:30:34 +0100
Subject: no Werror

Remove -Werror flag and do not treat warnings as errors anymore. This is a
useful development flag but will cause a FTBFS whenever GCC becomes more
strict.

Bug-Debian: https://bugs.debian.org/897760
Forwarded: no
---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 86af0cb..82e5a4e 100644
--- a/configure.in
+++ b/configure.in
@@ -23,7 +23,7 @@ AC_ARG_ENABLE(warn,
 AC_HELP_STRING([--enable-warn],[Enable compiler warnings @<:@default=on@:>@]),
 	enable_warn=$enableval, enable_warn=yes)
 if test "x$enable_warn" = xyes; then
-	CFLAGS="$CFLAGS -Wall -Werror"
+	CFLAGS="$CFLAGS -Wall"
 fi
 
 AC_ARG_ENABLE(debug,
